components/ai_chat/common/buildflags/BUILD.gn

```diff
@@ -12,4 +13,11 @@ buildflag_header("buildflags") {
     "ENABLE_AI_CHAT=$enable_ai_chat",
     "BRAVE_AI_CHAT_ENDPOINT=\"$brave_ai_chat_endpoint\"",
   ]
+
+  if (is_android &&
+      (brave_channel == "nightly" || brave_channel == "development")) {
```

Comment:
just an FYI that this might be conflicting with another change from @petemill because I believe he will also check for  `|| brave_channel == "beta" ||| brave_channel == "dev"`

Reply:
I think we will need to add another condition `else if (!is_android && (...))` in that case in the feature PR, @petemill let me know do you want me to add it? Also I think we have it enabled on desktop nightly via griffin. Do we want to remove it from griffin in that case or just add beta there?